plugin.go

```diff
@@ -1,12 +1,24 @@
 package plugin
 
 import (
-	"github.com/corazawaf/coraza/v2"
-	"github.com/corazawaf/coraza/v2/operators"
-	"github.com/corazawaf/coraza/v2/seclang"
+	"github.com/corazawaf/coraza/v3/experimental/plugins"
+	"github.com/oschwald/geoip2-golang"
 )
 
-func init() {
-	operators.RegisterPlugin("geoLookup", func() coraza.RuleOperator { return new(geoLookup) })
-	seclang.RegisterDirectivePlugin("secGeoLookupDb", directiveSecGeoLookupDb)
+func RegisterGeoDatabase(database []byte, databaseType string) {
+	db, err := geoip2.FromBytes(database)
+	if err != nil {
+		panic(err)
```

Comment:
not sure about this panic. I believe we don't want to turn down the platform.

Reply:
Fair point, I return the error now, and leave the handling up to the caller.